pki.spec

```diff
@@ -816,7 +811,7 @@ cd build
     -DVERSION=%{version}-%{release} \
     -DVAR_INSTALL_DIR:PATH=/var \
     -DP11_KIT_TRUST=/etc/alternatives/libnssckbi.so.%{_arch} \
-    -DJAVA_HOME=%{java_home} \
+    -DJAVA_HOME=%java_home \
```

Comment:
Is `%java_home` part of the RPM macro? I see that you removed line#52.  Can it stay `%{java_home}`, to be uniform with other params?

Reply:
`%java_home` is provided by `javapackages-tools` and is a RPM macro, yes.

Reply:
I've filed a BZ to figure out why tests fails:

https://bugzilla.redhat.com/show_bug.cgi?id=1830340